repos/rust-lang/rust-artwork.toml

```diff
@@ -0,0 +1,6 @@
+org = "rust-lang"
+name = "rust-artwork"
+description = "Official artwork for the Rust project."
+bots = []
+
+[access.teams]
```

Comment:
Can we leave a comment about why the access is empty so that if someone is looking at this file they aren't confused.

Reply:
Ok, added a comment.